core/src/main/java/com/scalar/db/util/TableMetadataManager.java

```diff
@@ -1,12 +1,11 @@
-package com.scalar.db.storage.common;
+package com.scalar.db.util;
```

Comment:
I moved the `TableMetadataManager` class from the `com.scalar.db.storage.common` package to the `com.scalar.db.util` package because it's used not only the classes in the `storage` package but also the classes in the `transaction` package.